src/Pyz/Zed/Checkout/CheckoutDependencyProvider.php

```diff
@@ -48,6 +49,7 @@ protected function getCheckoutPreConditions(Container $container)
     {
         return [
             new CustomerPreConditionCheckerPlugin(),
+            new CustomerAddressSalutationCheckoutPreConditionPlugin(),
```

Comment:
No `CustomerAddressSalutationCheckoutPreConditionPlugin` here. Is this correct?

Reply:
Discussed, the diff is related to the guest cart feature which is absent in the b2b. All good here.